### PR TITLE
Added back function to incorporate a self-signed or otherwise bad cert

### PR DESCRIPTION
**Pull request by hugh-mend**

### CODE REVIEW

**Comment by hugh-mend** [comment]
Hi @samq-ws this self-signed-cert code got lost in the refactoring.  Also the code was broken due to some changes in how the images were built; I think I've fixed that.

**Comment by randygeyer-ws** [comment]
There's a better way to incorporate certs into the containers that requires no DockerFile changes.  I don't support this method.

File: Repo-Integration/Self-Hosted/setup.sh [hunk 1]
```diff
@@ -77,6 +81,64 @@ echo "${grn}Download Success!!!${end}"
 
 }
 
+function cert_add(){
```
Comment (randygeyer-ws) [inline Repo-Integration/Self-Hosted/setup.sh:84]: There's a better way to incorporate certs into the containers that require no changes to the dockerfiles.  I don't support this approach.

**Review by randygeyer-ws — COMMENTED** [review]
There's a better way to incorporate certs into the containers that require no changes to the dockerfiles. I don't support this approach.

**Review by samq-ws — CHANGES_REQUESTED** [review]
@hugh-mend , please meet with @randygeyer-ws offline and find a way to document his better approach for customers.

**Comment by samq-ws** [comment]
Please see [this addition](https://github.com/mend-toolkit/mend-examples/blob/main/Repo-Integration/Self-Managed/certs.md)

**Pull request closed without merging**